ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,61 @@
+
+- [ ] Documentation is complete. [Documentation writing info](#)
+    - Read through documentation and make sure the following is done
+      - [ ] Description of component or pattern
+      - [ ] HTML code snippet example(s)
+      - [ ] React code snippet example(s)
+      - [ ] All react properties are explained
+      - [ ] Guidance on when to use this item
+      - [ ] Guidance on when not to use this item
+      - [ ] Accessibility considerations
+
+---
+
+- [ ] Validate HTML
+  - Go to [validator.w3.org](https://validator.w3.org/#validate_by_input) paste your code in and make sure it validates without error.
+
+---
+
+- [ ] Keyboard testing
+  - Navigate the item using your Keyboard
+  - What to check:
+    - [ ] Navigate using **tab (navigate forward)** and **shift + tab (navigate backward)**
+      - Keyboard focus indicators are present
+      - Navigation order is logical and intuitive
+    - [ ] Links - Follow links by using **enter**
+    - [ ] Buttons - use **enter** or **spacebar** to click a button
+    - [ ] Checkbox - **spacebar** to check/uncheck
+    - [ ] Radio buttons - **up/down** arrows or **left/right** arrows to select an option
+    - [ ] Dropdowns - **up/down** arrows to navigate between options. **Spacebar** to open close select
+    - [ ] Autocomplete - **Type** to begin filtering. **up/down** arrows to navigate between options. **Enter** to select an option.
+    - [ ] Dialog/modal - **Esc** should close the window
+      - Modal dialogs should maintain keyboard focus.
+      - When a dialog closes, focus should usually return to the element that opened the dialog.
+    - [ ] Tabs - **Tab** once to navigate into the group of tabs and once to navigate out of the group of tabs. **up/down** arrows or **left/right** arrows to move through tabs.
+    - [ ] Scroll - **up/down** arrows to scroll vertically. **Left/right** arrows to scroll horizontally. **Spacebar** scroll down or **shift + spacebar** to scroll up by page.
+
+---
+
+- [ ] Color blindness checks
+  - View the component to check for visual issues. If using Google Chrome [Colorblinding](https://chrome.google.com/webstore/detail/colorblinding/dgbgleaofjainknadoffbjkclicbbgaa?hl=en) or [ChromeLens](https://chrome.google.com/webstore/detail/chromelens/idikgljglpfilbhaboonnpnnincjhjkd/related?hl=en) are great plugins.
+  - What to check:
+      - [ ] Red-Blind / Protanopia
+      - [ ] Green-Blind / Deuteranopia
+      - [ ] Blue-Blind / Tritanopia
+      - [ ] Red-Weak / Protanomaly
+      - [ ] Green-Weak / Deuteranomaly
+      - [ ] Blue-Weak / Tritanomaly
+      - [ ] Monochromacy / Achromatopsia
+      - [ ] Blue Cone Monochromacy / Achromatomaly
+
+---
+
+- [ ] 400% Zoom
+  - Set your browser to 1280px width, and scale up to 400%
+  - What to check for:
+      - [ ] Still usable at 400% increase in size
+      - [ ] Content does not overlap or force users to scroll horizontally
+      - [ ] All content is available
```

Comment:
I'd like to add a section for Screen Reader testing also. This will need language for users to test themselves, or to have us test, if they don't have access to the necessary equipment. I don't expect most folks who are contributing to have a full JAWS license at $1,000+, but if they have tested with open-source or OS readers like NVDA or VoiceOver, that'll be great.

If contributors need us to do this testing, they could maybe mark a box or leave the checkboxes blank?

Screen readers and browsers I'd like to consider for passing criteria:
* Windows: IE11 + JAWS
* Windows: NVDA + Firefox
* MacOS: Safari + VoiceOver

Bonus testing:
* Windows: Chrome + JAWS
* iOS: Safari + VoiceOver
* Android: Chrome + Talkback

Reply:
I started some documentation here https://github.com/adhocteam/a11y-snippets/blob/sw-screen-reader-set-up/worksheets/how-to-set-up-screen-readers.md